### ISSUE TITLE
Conflict with Elementor and Jetpack Swiper Library

### ISSUE THREAD

**Issue by bobmatyas** [Automattic/jetpack#20579]
Elementor support has reported that there is a conflict with the Swiper styles in Jetpack versions v9.9 and up (including v9.9.1 and v10).

#### Steps to reproduce the issue

A user reported the following to Elementor:

> Our catalogue which has been set to show 1 column on mobile view has been displaying 2 columns which makes product window too thin, hence shows all details garbled-up, so I contacted Elementor as I use their plugin to set the number of columns which is set to 1 column.

Elementor responded with:

> I understood the case and have solution for this. The issue is not due to Elementor but Jetpack latest version 9.9 conflicting with swiper library Elementor certain widget uses. If you roll back to Jetpack v9.8.1, issue will not happen. You can download it by going to the below URL: https://wordpress.org/plugins/jetpack/advanced/

> For now, either you can roll back Jetpack or use this below temporary CSS code in affected widget custom CSS area to fix the issue -

```
/* Temporary fix for Jetpack CSS conflict with Elementor Slides & Carousel widgets */
selector .swiper-container{
width: 100%;
}
```

Reported on 4197462-zen

**Comment by bobmatyas** [comment]
Not sure if this is related, but there have been 2 other reports of post v9.9.1 issues with Swiper:

* https://github.com/Automattic/jetpack/issues/20289#issuecomment-884725575
* https://github.com/Automattic/jetpack/issues/20289#issuecomment-884947043

**Comment by sdixon194** [comment]
Got another report here, but without Elementor: https://github.com/Automattic/jetpack/issues/20581

@glendaviesnz I know you worked on the initial fix for this in 9.9.1. Would you be able to take a peek and see if anything comes to mind that might be going on with swiper overriding theme styles still?

I tried reproducing on Elementor sites, but not sure if I'm testing correctly, everything still worked correctly.

**Comment by glendaviesnz** [comment]
@sdixon194 I have put up a PR at https://github.com/Automattic/jetpack/pull/20599 which adds a jetpack selector to all of the default swiper css. I think this is going to be the only robust way to solve this issue.
